Adafruit_NeoPixel.cpp

```diff
@@ -1544,6 +1548,7 @@ void Adafruit_NeoPixel::show(void) {
   }// End of DMA implementation
   // ---------------------------------------------------------------------
   else{
+#ifdef digitalPinToBitMask     

```

Comment:
this is a bad ifdef technique, please use ARDUINO_ARCH_NRF52840 or similar

Reply:
Fixed with ARDUINO_ARCH_NRF52840.